pom.xml

```diff
@@ -431,6 +431,12 @@
                                 <goals>
                                     <goal>sign</goal>
                                 </goals>
+                                <configuration>
```

Comment:
Had to add this to pick up the passphrase for the gpg key that is set in the setup java step